### PR TITLE
Topk fixes

### PR DESCRIPTION
**Pull request by jithunnair-amd**

### CODE REVIEW

File: aten/src/THC/THCAsmUtils.cuh [hunk 1]
```diff
@@ -106,8 +104,8 @@ __device__ __forceinline__ unsigned getLaneMaskLt() {
 }
 
 #if defined (__HIP_PLATFORM_HCC__)
-__device__ __forceinline__ unsigned long long int getLaneMaskLe() {
-  std::uint64_t m = (1ull << (getLaneId() + 1ull)) - 1ull;
+__device__ inline unsigned long long int getLaneMaskLe() {
```
Comment (iotamudelta) [inline aten/src/THC/THCAsmUtils.cuh]: Given how `__forceinline__` is now correctly defined, I'd like to keep it that way

**Comment by iotamudelta** [comment]
@jithunnair-amd do you want to make that change or shall I?

**Comment by jithunnair-amd** [comment]
I'll put it in. Thanks for spotting!

**Comment by iotamudelta** [comment]
weird failure in last CI run:

test_sparse_add_coalesce (__main__.TestCudaSparse) ... Memory access fault by GPU node-4 (Agent handle: 0x1879630) on address 0x14000. Reason: Page not present or supervisor privilege.
19:04:41 Traceback (most recent call last):
19:04:41   File "test/run_test.py", line 389, in <module>
19:04:41     main()
19:04:41   File "test/run_test.py", line 381, in main
19:04:41     raise RuntimeError(message)

**Pull request merged**